docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -30,24 +30,24 @@ node to the selected remote _gateway nodes_ only.
 The _gateway nodes_ selection depends on the following criteria:
 
 - *version*: Remote nodes must be compatible with the cluster they are
-registered to. This is subject to the same rules as <<rolling-upgrades>>.
-Any node can communicate with any other node on the same major version (e.g.
-6.0 can talk to any 6.x node). Only nodes on the last minor version of a
-certain major version can communicate with nodes on the following major
-version (e.g. 6.7 can communicate with 7.0, as well as any 7.x node, while
- 6.6 or earlier cannot talk to any 7.x node). Note that version compatibility
- is symmetric, meaning that if 6.7 can communicate with 7.0, 7.0 can also
- communicate with 6.7. The matrix below summarizes compatibility as described
- above.
+registered to. This is subject to rules that are similar to those for
+<<rolling-upgrades>>. Any node can communicate with any other node on the same
+major version (e.g. 7.0 can talk to any 7.x node). Only nodes on the last minor
+version of a certain major version can communicate with nodes on the following
+major version. Note that in the 6.x series, both 6.7 and 6.8 can communicate
```

Comment:
I think this is still incorrect. I don't think a 7.1 node will communicate with a 6.7 node, and we certainly don't test it. To upgrade across majors to 7.1, a user must come from 6.8. Version.minimumCompatibilityVersion() looks at the version constants and finds the previous major's last minor.

Reply:
I tested it, and you are right 6.7 can't talk to 7.1.